sagemaker-pipelines/time_series_forecasting/amazon_forecast_pipeline/sm_pipeline_with_amazon_forecast.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    from sagemaker.workflow.parameters import (  # figure out if paraminteger works for forecast_horizon
</pre><p>Resolve this comment, it looks like a TODO item.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>